### DISCUSSION TITLE
Moved old messages from another folder into inbox sorted wrong and messages come top of the list

### DISCUSSION THREAD

**Discussion by Hanswax** [the-djmaze/snappymail#1165] [Q&A]
Hi everyone,

i have a problem with Snappy Mail.

When i move an old massage from inbox to a folder and move it back to inbox, message comes on top although the message is an old message.

![image](https://github.com/the-djmaze/snappymail/assets/135695119/90c447b1-b41b-4a57-882c-30cea152d8ba)

Thanks !

**Comment by cm-schl** [comment]
Hi, I can't reproduce this on v2.28.1.
How do you sort your mails?

**Reply by Hanswax** [reply]
Hi, can u please try to move an old mail to an Archiv and move it back to inbox again?

i sort the mails by date.

Thanks !

**Reply by cm-schl** [reply]
Same here - but the mail gets correctly resorted to its old place...
in your screenshot I see some mails with a red mark - has the mail a "important" tag before or after you move it?

**Reply by Hanswax** [reply]
they marked themselves after i moved them back from another folder.

**Reply by cm-schl** [reply]
I tried to move a mail to different folders (also archive) by drag&drop and by the menu button, but this does not happen on my install... The fact that the mail is marked when it gets moved to the inbox is interesting. Do you use your own IMAP server or some freemail account?

**Reply by Hanswax** [reply]
i use my own IMAP Server and it not happend in IMAP server.

**Reply by cm-schl** [reply]
I think that's a case for @the-djmaze 🤔🙂

**Reply by Hanswax** [reply]
Yeap i hope he can be helpfull 🙄

Thanks for ur help

**Reply by the-djmaze** [reply]
This is normal behavior for IMAP servers that do not fully support IMAP4rev1
You can read #1022 and #1062
